.github/workflows/codeql-analysis.yml

```diff
@@ -2,9 +2,9 @@ name: 'codeql'
 
 on:
   push:
-    branches: [main]
+    branches: [main, v10]
   pull_request:
-    branches: [main]
+    branches: [main, v10]
```

Comment:
Would you want to just turn this off for the v10 branch?

Reply:
🤷 I don't mind leaving it on for peace of mind?